src/components/App.tsx

```diff
@@ -881,6 +884,13 @@ export default class App extends React.Component<any, AppState> {
     });
   }
 
+  setLocalPMTiles = (file: File) => {
+    const pmt = new PMTiles(new FileSource(file));
```

Comment:
I'm not sure if this should be here on right after selecting the file and raising an event with the selected `PMTiles` class.
Either way this is fine. your call.